docs/config/io_helidon_common_configurable_AllowList.adoc

```diff
@@ -0,0 +1,59 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+ifndef::rootdir[:rootdir: {docdir}/..]
+:description: Configuration of io.helidon.common.configurable.AllowList
+:keywords: helidon, config, io.helidon.common.configurable.AllowList
+:basic-table-intro: The table below lists the configuration keys that configure io.helidon.common.configurable.AllowList
+include::{rootdir}/includes/attributes.adoc[]
+
+= AllowList (common.configurable) Configuration
+
+// tag::config[]
+
+
+Type: link:{javadoc-base-url}/io.helidon.common.configurable/io/helidon/common/configurable/AllowList.html[io.helidon.common.configurable.AllowList]
+
+
+
+
+== Configuration options
```

Comment:
my preference is title case but I see sentence case used throughout. H2 Configuration Options H3/H4 Configuration options. I'm ok with either as long as we're consistent (and we aren't across the board).

Reply:
These `.adoc` files are generated by a separate tool Tomas wrote. The tool could emit whatever we agree makes sense.